app/controllers/tags_controller.rb

```diff
@@ -133,7 +133,7 @@ def merge
     posts_sql = 'UPDATE posts INNER JOIN posts_tags ON posts.id = posts_tags.post_id ' \
                 'SET posts.tags_cache = REPLACE(posts.tags_cache, ?, ?) ' \
                 'WHERE posts_tags.tag_id = ?'
-    exec([posts_sql, "\n- #{@subordinate.name}", "\n- #{@primary.name}", @subordinate.id])
+    exec([posts_sql, "\n- #{@subordinate.name}\n", "\n- #{@primary.name}\n", @subordinate.id])
```

Comment:
Since exec is quite a dangerous command to run, I would be in favor of renaming the function to something that is at least not a ruby builtin accessible at all locations. Something like `query` or `exec_query` would probably be a better name.

Reply:
Sounds sensible to me. @Taeir do you want to do that in this PR or merge this and do that separately?

Reply:
I've updated the PR with the new name for `exec` (`exec_sql`)